account_statement_import_txt_xlsx/readme/USAGE.rst

```diff
@@ -0,0 +1,4 @@
+To use this module, you need to:
+
+#. Get statement in TXT/CSV or XLSX from your online banking software
+#. Go to Odoo and and import the statement file, selecting corresponding format
```

Comment:
It would be helpful to have some extra information showing how to import statement files into Odoo.